### PR TITLE
Add UMD builds

### PR DESCRIPTION
**Pull request by MayhemYDG**
I've also tweaked the cells map in the Row component, as promised.

### CODE REVIEW

File: packages/react-data-grid/src/Row.tsx [hunk 1]
```diff
@@ -47,7 +47,11 @@ export default class Row extends React.Component<RowRendererProps> implements Ro
 
     const cellProps: CellRendererProps & { ref: (cell: CellRenderer | null) => void } = {
       ref: (cell) => {
```
Comment (qili26) [inline packages/react-data-grid/src/Row.tsx]: Do we have any preference? Your code vs this
```
(cell) => cell ? this.cells.set(key, cell) : this.cells.delete(key);
```

Comment (MayhemYDG) [reply packages/react-data-grid/src/Row.tsx]: Both are good IMO.

File: jest.config.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-'use strict';
+
```
Comment (qili26) [inline jest.config.js]: Any reason removing the `use strict`? I believe if we remove this, we don't need the ln#1 and 2 anymore.

Comment (MayhemYDG) [reply jest.config.js]: Oops, fixed it now. I need to adjust the eslint config so it doesn't autofix it away.

**Pull request merged**